### PR TITLE
[sunglass_hut] refactor to single spider (6270 locations), remove obsolete lenscrafters, pearle_vision, target_optical_us

### PR DESCRIPTION
**Pull request by arrival-spring**
Turns out there is a single Yext API for Sunglass Hut, LensCrafters, Pearle Vision and Target Optical.

Downside is that some of the claimed websites of the form https://stores.sunglasshut.com/* don't work in certain countries (those that were covered by other spiders:.sunglass_hut_2, sunglass_hut_3, sunglass_hut_latam). Perhaps they are in the process of transitioning to this?

~~More countries are now on the sitemap in sunglass_hut_1 and now collecting coordinates there~~

~~Refactor sunglass_hut_2~~

~~Add sunglass_hut_3 for some more countries~~

~~Add sunglass_hut_na_za~~

~~Add sunglass_hut_in~~

~~Add sunglass_hut_latam~~

~~Remove sunglass_hut_gb as it was broken and gb is covered by sunglass_hut_1~~

### CODE REVIEW

**Comment by arrival-spring** [comment]
Is bot protection the reason those two are failing?

Both work fine locally

File: locations/google_url.py [hunk 1]
```diff
@@ -94,7 +94,10 @@ def get_query_param(link, query_param):
 
     if "/maps.google.com/" in url:
         for ll in get_query_param(url, "ll"):
-            lat, lon = ll.split(",")
+            lat_lon = ll.split(",")
+            if len(lat_lon) == 3:  # Zoom value can be included in ll options
```
Comment (iandees) [inline locations/google_url.py:98]: There's an existing test suite for google_url. Can you add a case for this please?

Comment (arrival-spring) [reply locations/google_url.py:98]: Sure, have done with one such url from one of these pages

**Comment by arrival-spring** [comment]
Draft because sunglass_hut_1 has a yext API which needs research as it returns many extra results

**Comment by arrival-spring** [comment]
See updated comment at the top

**Pull request merged**